Lib/test/test_descr.py

```diff
@@ -1303,6 +1303,11 @@ class X(object):
         with self.assertRaises(AttributeError):
             del X().a
 
+        class X(object):
```

Comment:
Why inherit from object?

Reply:
To cover also some obscure backwards compat code paths. I can remove if you want

Reply:
No big deal for me, but I guarantee some over-zealous "code cleaner" is going to come along some day and remove it, so I'd add a comment if you want it to stick around.